R/bayesian.R

```diff
@@ -575,3 +603,86 @@ hpdi_single <- function(samples, interval, reportvars, weighted, lp)
                        paste(interval,'|'))
     mat
 }
+
+#' Recalculate the posterior PDFs for a set of saved scenarios
+#'
+#' Load a set of scenario-info files generated by an ensemble run,
+#' recalculate their posterior probability densities, and optionally save the
+#' results.
+#'
+#' Normally the reason you would do this would be to change the prior
+#' and/or likelihood functions used in the analysis, or to add those
+#' calculations to scenario outputs where they were never calculated in the
+#' first place.  Either way, it only makes sense to do this with uniformly
+#' sampled, posterior-weighted samples.  If you do this with Markov Chain Monte
+#' Carlo samples, the recorded likelihood and posterior probability values will
+#' change, but the sampling will still reflect the old probability density.
+#'
+#' tldr: \strong{Do not use this function on the results of a Markov Chain Monte
+#' Carlo analysis!}  If you do, not only will you get bogus results, you will
+#' bring shame and disgrace on yourself and everyone you have ever worked with.
+#' Don't do it.
```

Comment:
well, this is a little direct

Reply:
It's maybe a little over the top.  I'm not sure whether this function really belongs in the final product or not.  I originally developed it to go back and do the probability density calculations on a bunch of runs where we had initially done just the model runs.  It's not clear that it has a use going forward, but it's also not clear that we _won't_ want to use it again.  Hence the dire warnings about not using it unless you're sure it's appropriate.

I can do some cleanup on this function (and its docs), or we can just remove it entirely.  I guess a middle way would be to demote it to a script and include it in `inst/scripts`.  What do you think, @kvcalvin?  Keep it or cut it?

Reply:
Let's move it to scripts if it isn't critical for the package

Reply:
Ok, done.